src/main/java/net/snowflake/client/jdbc/SnowflakeDatabaseMetaData.java

```diff
@@ -1127,8 +1131,18 @@ public ResultSet getProcedureColumns(
     while (resultSetStepOne.next()) {
       String procedureNameUnparsed = resultSetStepOne.getString("arguments").trim();
       String procedureNameNoArgs = resultSetStepOne.getString("name");
+      String schemaName = resultSetStepOne.getString("schema_name");
```

Comment:
Now that results are getting correctly returned with wildcards, we have to narrow down the results so that only results matching the input schema and procedure name are returned.

Reply:
Hi @sfc-gh-mknister 

I'm trying to fix https://github.com/snowflakedb/snowflake-jdbc/issues/550 this issue and now I have question - what about lower case schema here? Like "lcase" as example. If schemaPattern == "lcase" and schemaName ""lcase"" ->
`compiledSchemaPattern.matcher(schemaName).matches()` will be false, so I can't get my procedure columns. Maybe any advice how to fix this here?